GSoC26/src/dbpedia_linker.py

```
@@ -0,0 +1,74 @@
+    # 2. Resolve Slang (Using the imported function!)
+    clean_query = resolve_redirect(query)
+
+    # 3. Candidate Generation
+    candidates = [clean_query]
+    if "fc" not in clean_query.lower():
+        candidates.append(clean_query + " FC")
+
+    all_matches = []
+    for term in candidates:
+        try:
+            r = requests.get(DBPEDIA_LOOKUP, params={"query": term, "format": "json", "MaxHits": 20}).json()
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)